ui/app/registration/views/patientcommon.html

```diff
@@ -278,6 +278,24 @@
         <div patient-relationship patient="patient" ng-show="patient.hasRelationships"
              ng-if="::relationshipTypes"></div>
     </div>
+    <div class="box-container">
+        <legend class="additional-attribute" toggle="patient.hasIndexInfo">
+          <span>
+              <i class="fa fa-caret-right"></i>
+              <i class="fa fa-caret-down"></i>
+              <strong><a ng-bind-html="::'REGISTRATION_TITLE_INDEX_TESTING_CONTACT_INFO' | translate"></a></strong>
+          </span>
+        </legend>
+
```

Comment:
@tmvumbi2 Please add some space between the title and the question, and move the indent to the right (as in other information).

![index](https://user-images.githubusercontent.com/1198827/74811722-c514d580-52fa-11ea-85ab-84903d70bdf6.png)